typings/number_format.d.ts

```diff
@@ -34,7 +34,7 @@ declare module "react-number-format" {
     mask?: string | string[];
     value?: number | string;
     isNumericString?: boolean;
-    customInput?: (inputComponent: React.ComponentType<any>) => void;
+    customInput?: React.ReactElement,

```

Comment:
customInput is a React component class or a function reference (ex : TextField). `React.ReactElement` will be valid for instance (ex: <TextField />). 
Though `(inputComponent: React.ComponentType<any>) => void;` is also not correct